tests/data_files/Makefile

```diff
@@ -1131,6 +1131,16 @@ cert_md5.crt: cert_md5.csr
 	$(MBEDTLS_CERT_WRITE) request_file=$< serial=6 issuer_crt=$(test_ca_crt) issuer_key=$(test_ca_key_file_rsa) issuer_pwd=$(test_ca_pwd_rsa) not_before=20000101121212 not_after=20300101121212 md=MD5 version=3 output_file=$@
 all_final += cert_md5.crt
 
+################################################################
+#### Diffie-Hellman parameters
+################################################################
+
+dh.998.pem:
+	$(OPENSSL) dhparam -out $@ -text 998
```

Comment:
I dont' quite understand this. Isn't that generating the dh.998.pem file, the one that is also added in this PR?

Reply:
Yes, that's by design. See the comment at the top of the makefile.

Reply:
ooooh. Ok, that makes sense now, thank you :)